kibbeh/src/stories/UserBadge.story.tsx

```diff
@@ -9,14 +9,14 @@ export default {
 
 const TheUserBadge: Story = () => {
   return (
-    <div className="flex flex flex-row">
-      <div className="flex m-1">
+    <div className="flex flex-row gap-1">
```

Comment:
I believe we no longer use gap because we had some cross browser related issues. (@overlisted could you confirm?)

Reply:
yeah just use margins until i make the polyfill ready for tailwind

Reply:
i'll fix that to include margin.